CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 
 ## <next>
 
+## 9.1.12
```

Comment:
Rebase or merge latest master changes and update changelog version to 9.1.13 since there's already 9.1.12 published.